swarm/bmt/bmt.go

```diff
@@ -318,7 +320,7 @@ func (h *Hasher) Sum(b []byte) (s []byte) {
 // with every full segment calls writeSection in a go routine
 func (h *Hasher) Write(b []byte) (int, error) {
 	l := len(b)
-	if l == 0 || l > 4096 {
+	if l == 0 || l > chunk.DefaultSize {
```

Comment:
Type mismatch: `l` is int and `chunk.DefaultSize` is int64.

Reply:
Fixed